README.md

```diff
@@ -1,13 +1,19 @@
 # stackup2-barefoot-frontend
 
-[![Build Status](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend.svg?branch=develop)](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend)
```

Comment:
Why did you remove this build status badge for here? In my opinion, I think the best place for badges should be this straight below to the project's title